eiffel-gerrit-plugin-script

```diff
@@ -0,0 +1,138 @@
+#! /bin/bash
+
+GERRIT_API_HOST=""
+STATUS=0
+
+function invalid_arguments {
+  echo "missing command(s) start | stop | test | only_test | restart | build | rebuild"
+  echo "you may combine any commands in the order you wish to execute them"
+  echo " "
+  echo " start     :: start the docker containers"
+  echo " stop      :: stop the docker containers"
+  echo " test      :: build, start, test and stop docker containers"
+  echo " only_test :: executes the test, docker containers must be running"
+  echo " restart   :: restarts the docker containers"
+  echo " build     :: will build the docker images to be used"
+  echo " rebuild   :: see --> build"
+  echo " "
+  echo "Example:"
+  echo "./eiffel-gerrit-plugin-script test"
+  echo "      --> Will compile the source code to an executable jar and "
+  echo "          Build the docker file and start the docker-compose"
+  echo "          Execute the tests and then stop the running services"
+
+  exit;
+}
+
+function do_stop {
+  echo "########################################"
+  echo "###### STOPING DOCKER SERVICES #########"
+  echo "########################################"
+  docker-compose -f target/docker-compose.yml down
+}
+
+function do_rebuild {
+  echo "########################################"
+  echo "######### BUILDING JARS ################"
+  echo "########################################"
+
+  mvn clean install package -DskipTests
+
+  mv target/test-classes/docker-compose.yml target/docker-compose.yml
+  mv target/test-classes/Dockerfile target/Dockerfile
+
+  docker build -f target/Dockerfile -t plugin-test target/
+}
+
+function do_start {
+  echo "########################################"
+  echo "###### STARTING DOCKER SERVICES ########"
+  echo "########################################"
+
+  docker-compose -f target/docker-compose.yml up -d
+
+  GERRIT_CONTAINER_ID=$(docker ps | grep target_gerrit | awk '{print $1}')
+  GERRIT_API_HOST=$(docker inspect -f '{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}' $GERRIT_CONTAINER_ID)
+
+  GERRIT_BASE_URL=http://$GERRIT_API_HOST:8080
+
+  printf "WAITING FOR GERRIT TO START WITH URL: "$GERRIT_BASE_URL
+  echo " "
+
+  attempt_counter=0
+  max_attempts=60
+
+  until $(curl --output /dev/null --silent --head --fail $GERRIT_BASE_URL); do
+      if [ ${attempt_counter} -eq ${max_attempts} ];then
+        printf " \e[91m FAILURE! \e[39m"
+        echo " "
+        echo "Gerrit failed to start aborting!"
```

Comment:
Add a full stop between start and aborting, otherwise it reads very funny ... :)

Reply:
Done.